tests/unit_tests/db/test_db_handler.py

```diff
@@ -347,6 +347,18 @@ def test_update_parameter_field_db(db, random_id, db_cleanup, io_handler):
     )
     assert pars["camera_pixels"]["applicable"] is False
 
+    with pytest.raises(ValueError):
+        db.update_parameter_field(
+            db_name=f"sandbox_{random_id}",
```

Comment:
With the other big PR merged we have now to replace the DB to the Staging one, right? We should remind both Orel and Tobias to do that as well once they return.